pkg/rtc/participant.go

```diff
@@ -1372,7 +1394,22 @@ func (p *ParticipantImpl) addPendingTrack(req *livekit.AddTrackRequest) *livekit
 		Source:     req.Source,
 		Layers:     req.Layers,
 	}
-	p.pendingTracks[req.Cid] = &pendingTrackInfo{TrackInfo: ti}
+	pendingInfo := &pendingTrackInfo{TrackInfo: ti}
+	for _, codec := range req.SimulcastCodecs {
+		mime := codec.Codec
```

Comment:
is this done to fix clients that are only passing in `h264` or `av1` ? I think we should reject instead of fixing.. [MIME types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) should always be in `type/subtype` syntax

Reply:
normalize mime type here, since client only send codec name. remove this need every client send standard MIME type